backend/src/main/java/gov/cdc/usds/simplereport/service/CsvExportService.java

```diff
@@ -0,0 +1,525 @@
+              params.startDate(),
+              params.endDate(),
+              pageable.getPageNumber(),
+              pageable.getPageSize())
+          .map(TestResultsListItem::new);
+    }
+  }
+
+  private CSVFormat createCsvFormat() {
+    String[] headers = {
+      "Patient first name",
+      "Patient middle name",
+      "Patient last name",
+      "Patient full name",
+      "Patient date of birth",
+      "Test date",
+      "Condition",
+      "Result",
+      "Result reported date",
+      "Test correction status",
+      "Test correction reason",
+      "Device name",
+      "Device manufacturer",
+      "Device model",
+      "Device swab type",
+      "Has symptoms",
+      "Symptoms present",
+      "Symptom onset",
+      "Facility name",
+      "Submitter",
+      "Patient role",
+      "Patient ID (Student ID, Employee ID, etc.)",
+      "Patient preferred language",
+      "Patient phone number",
+      "Patient email",
+      "Patient street address",
+      "Patient street address 2",
+      "Patient city",
+      "Patient state",
+      "Patient zip code",
+      "Patient county",
+      "Patient country",
+      "Patient gender",
+      "Patient race",
+      "Patient ethnicity",
+      "Patient tribal affiliation",
+      "Patient is a resident in a congregate setting",
+      "Patient is employed in healthcare"
+    };
+    return CSVFormat.Builder.create().setHeader(headers).build();
+  }
+
+  private void writeCsvRow(CSVPrinter csvPrinter, TestResultsListItem item) throws IOException {
+    Person patient = item.getPatient();
+    DeviceType deviceType = item.getDeviceType();
+    Facility facility = item.getFacility();
+    AskOnEntrySurvey surveyData = item.getSurveyData();
+    ApiUser createdBy = item.getCreatedBy();
+
+    String firstName = extractString(patient, Person::getFirstName);
+    String middleName = extractString(patient, Person::getMiddleName);
+    String lastName = extractString(patient, Person::getLastName);
+    String fullName = formatFullName(firstName, middleName, lastName);
+
+    String submitterName = "";
+    if (createdBy != null && createdBy.getNameInfo() != null) {
+      submitterName =
+          formatFullName(
+              createdBy.getNameInfo().getFirstName(),
+              createdBy.getNameInfo().getMiddleName(),
+              createdBy.getNameInfo().getLastName());
+    }
+
+    String facilityName =
+        facility != null && facility.getFacilityName() != null
+            ? formatFacilityName(facility.getFacilityName(), facility.getIsDeleted())
+            : "";
+
+    String hasSymptoms = "";
+    if (surveyData != null) {
+      hasSymptoms =
+          formatHasSymptoms(
+              surveyData.getNoSymptoms() != null ? surveyData.getNoSymptoms() : false,
+              surveyData.getSymptomsJSON());
+    }
+
+    String symptomsPresent = "";
+    if (surveyData != null) {
+      symptomsPresent = formatSymptomsPresent(surveyData.getSymptomsJSON());
+    }
```

Comment:
I think this still needs to convert the SNOMEDs into the string name of the symptom if we want to match the existing behavior